### ISSUE TITLE
Invites: BUG and UX issue for incorrect email format when inviting

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1231]
**Insight from Round 1 ideation UX tests**

When inviting me as co-creator, user presses enter after typing 'Becca' as thinks this will be enough (he has already entered my email previously and is expecting it to have saved). 

He gets notified that it's the wrong email format: 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/b3ddf552-b61e-4f49-959b-51bd55b113b2)

He then edits it in that box: 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/3b63f64e-b75c-41f1-8e72-823d5934a2f6)

There is no button other than delete, so he assumes that'll be enough and clicks 'Send invites'. But it doesn't work. 

So he copy and pastes it back into the search box. It does work, but the notification that it's wrong is still lingering there:

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/08e36d7c-be4b-4392-86f4-dc57f84acd5d)

**Suggestions**

Presumably the incorrect email field should have the same UX as the Search field above, for adding it once amended?

Can the error message be fixed so it disappears once the problem is no longer a problem? :)

**Comment by kevincrepin** [comment]
When I type "Becca" and press enter I do get it properly though:
![Screenshot 2024-06-14 at 10 51 18](https://github.com/citizenos/citizenos-fe/assets/26792281/f4b383e1-12f8-4522-8e2e-3582e4b3b20a)

**Comment by kevincrepin** [comment]
But I also was able to replicate nothing happening after pressing "send invites" after correcting a wrong mail format.

**Comment by BeccaMelhuish** [comment]
Hmm OK maybe this is mostly just a bug then? 

I was going to make a separate issue, but the user mentioned that "It would be really nice if the system remembers emails I've typed in before", as he was inviting me several times in the test and wasn't getting my email come up. But it sounds like for Kevin it does, so maybe that's just a connected bug? I find similar, that it only seems to find emails once I've finished typing the whole address in, even when I am inviting them very often, and so would expect it to be remembered. 

UX-wise though @kevincrepin, this bit seemed off to me though, as he should have been able to edit the incorrect email in the same field as it appeared in, not have to go back to the 'Search' box to correct it?  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/ab612b5e-90eb-4f0a-b0d2-d0003196a920)

**Comment by kevincrepin** [comment]
Yep, he should be able to edit it there. The instruction says so as well :) I think that's the actual bug here, after correcting it and pressing "send invites" it also didn't work for me.

And I guess the system only remembers emails once that person has registered an account, @ilmartyrk ?

**Comment by BeccaMelhuish** [comment]
@kevincrepin @ilmartyrk I just tried it myself and I think as long as the 'correction' field works exactly the same as the field above it, in order to get the corrected email down into the 'accepted' emails bit, then it should be good. Currently for me if you press enter it works, but that's only if you think to press it, which many wouldn't.  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/8eabd0e5-e8c4-415d-a1df-5dfec35d37da)

Re the remembering emails, in all cases (for the user in test and for me) they already have accounts, so I think it must be something else :)
